src/Uuid.elm

```diff
@@ -69,6 +70,7 @@ fromString : String -> Maybe Uuid
 fromString text =
     if isValidUuid text then
         Just <| Uuid <| String.toLower text
+
     else
         Nothing
 
```

Comment:
Just below, there is a reference to the Random.Pcg package. Suggestion for replacement:

```elm
{-| Random Generator for Uuids. Using this Generator instead of the generate
function let you create lists of Uuids, map them to other types etc.
-}
```